internal/eval/migrated_eval.go

```diff
@@ -0,0 +1,216 @@
+package eval
```

Comment:
nit: instead of migrated eval, we can use the name `store_eval`?

Reply:
Done. Thanks